File: server/data/models/PageElement.ts
class PageElement
{
    id: number;
    type: string;
    content: string;

    constructor(elementId: number, type?: string, content?: string)
    {
        this.id = elementId;
        this.type = type ?? 'p';
        this.content = content ?? '';
    }
}

export {PageElement};

File: Notes/Data/Models/PageElement.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Notes.Data.Models
{
    public class PageElement
    {

        public int Id { get; set; }
        public string Type { get; set; }
        public string Content { get; set; }

        public PageElement(int id = 0, string elementType = "p", string content = "")
        {
            Id = id;
            Type = elementType;
            Content = content;
        }
    }
}
